bindings/experimental/distrdf/python/DistRDF/Ranges.py

```diff
@@ -157,6 +128,51 @@ class TaskTreeEntries:
     trees_with_entries: Dict[str, int] = field(default_factory=dict)
 
 
+def get_balanced_ranges(nentries, npartitions, exec_id: ExecutionIdentifier):
+    """
+    Builds range pairs from the given values of the number of entries in
+    the dataset and number of partitions required. Each range contains the
+    same amount of entries, except for those cases where the number of
+    entries is not a multiple of the partitions.
```

Comment:
maybe more simply you can just say "The `nentries` are divided uniformly among the `npartitions`".